src/app/fdctl/run/topos/fd_firedancer.c

```diff
@@ -446,6 +446,43 @@ fd_topo_initialize( config_t * config ) {
     fd_topob_tile_in(  topo, "rpcsrv", 0UL, "metric_in",  "stake_out",    0UL, FD_TOPOB_UNRELIABLE, FD_TOPOB_POLLED   );
   }
 
+  /* For now the only plugin consumer is the GUI */
+  int plugins_enabled = config->tiles.gui.enabled;
+  if( FD_LIKELY( plugins_enabled ) ) {
+    fd_topob_wksp( topo, "plugin_in"    );
+    fd_topob_wksp( topo, "plugin_out"   );
+    fd_topob_wksp( topo, "plugin"       );
+
+    /**/                 fd_topob_link( topo, "plugin_out",   "plugin_out",   128UL,                                    8UL+40200UL*(58UL+12UL*34UL), 1UL );
```

Comment:
You don't need all of these links. Just the ones that you've added. I think `plugin_out` and `replay_plugi`, `gossip_plugi`?